apps/key-manager/src/components/layout/Header.tsx

```diff
@@ -23,29 +22,14 @@ const Header = async () => {
             {session.user.isAdmin && (
               <p className={"text-destructive text-xl select-none"}>ADMIN</p>
             )}
-            <DropdownMenuTrigger className={"focus:outline-none"}>
-              {session.user.image && session.user.email && (
-                <Image
-                  className={"rounded-full"}
-                  width={40}
-                  height={40}
-                  src={session.user.image}
-                  alt={session.user.email}
-                />
-              )}
+            <DropdownMenuTrigger asChild>
+              <Avatar>
+                <AvatarImage src={session.user.image ?? ""} alt={"avatar"} />
```

Comment:
An explicit `<AvatarFallback>` is needed after `<AvatarImage>` with the desired content.

Reply:
Docs: https://www.radix-ui.com/primitives/docs/components/avatar#fallback

Also consider `delayMs` to prevent fallback from appearing at first.